federation/cluster/common.sh

```diff
@@ -135,6 +135,30 @@ function create-federation-api-objects {
 	    fi
 	    sleep 5
 	done
+
+	# Poll until DNS record for federation-apiserver is propagated
+	# TODO(colhom): REQUIREMENT for any other providers which use a DNS name for routing to loadbalancers
+	# right now it's just AWS that does this.
+	which nslookup
+	set +o errexit
+	if [[ "$KUBERNETES_PROVIDER" == "aws" ]];then
```

Comment:
GCE (for instance) attaches a Public IP directly to the loadbalancer, so there's nothing to poll as we already have the answer.